### PR TITLE
Feat/badges

### PR DESCRIPTION
**Pull request by ajay-mandaviya**
Here the preview of badge componen
Screenshot from 2022-02-05 23-19-07
Screenshot from 2022-02-05 23-19-35
t

### CODE REVIEW

File: component/badges/badges.css [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+}
+.badge-success {
+  background-color: var(--success);
+  color: var(--text);
+}
+.badge-pill {
+  border-radius: 10px;
+}
+.badge-link {
+  text-decoration: none;
```
Comment (nerdyspook) [inline component/badges/badges.css:43]: you can declare text-decoration: none; globally for all links as default and then change it where u need text-decoration

**Review by nerdyspook — COMMENTED** [review]
It looks good

File: component/badges/badges.css (hunk 1 above)
Comment (ajay-mandaviya) [reply component/badges/badges.css:43]: all ready make it , but i don't want to make class name much bigger , that's why re-declared here

File: component/badges/badges.css [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  border-radius: 10px;
+}
+.badge-link:hover {
+  color: var(--dark);
+}
+
+.badge-container {
+  display: flex;
+  position: relative;
+  width: 100px;
```
Comment (deepak29-git) [inline component/badges/badges.css]: you can use rem for achieve responsiveness

**Review by deepak29-git — COMMENTED** [review]
Everything looks good.

**Pull request merged**